tests/sillyCentroid.py

```diff
@@ -23,6 +23,6 @@
 from _sillyCentroid import SillyCentroidAlgorithm, SillyCentroidControl, SillyTransform
 
 # Do not register SillyCentroid in plugins.py, as it's not part of meas_base
-wrapSimpleAlgorithm(SillyCentroidAlgorithm, Control=SillyCentroidControl,
+wrapSimpleAlgorithm(SillyCentroidAlgorithm, Control=SillyCentroidControl, name="testLib_SillyCentroid",
```

Comment:
I think this algorithm is just on the border of being simple enough that it might have saved time to reimplement it in pure Python.  But now that you've done it this way that's probably not worth it.

Reply:
Yeah, it took me a bit to sort out the various "testLib" references (and lack thereof), but this works fine. Having someone implement SillyCentroid (which is basically identical to NaiveCentroid) in python might be a useful training exercise for a newer developer, but is otherwise not worth it.